Automattic/wp-calypso
# composite-checkout: add Giropay payment method

#### Changes proposed in this Pull Request

This adds the Giropay Stripe redirect payment method to composite checkout.

#### Screenshots

Old checkout:

![old-giropay](https://user-images.githubusercontent.com/2036909/85460095-e4ee2b80-b570-11ea-8fea-f63c70351f89.png)

New checkout:

Unselected:

![Screen Shot 2020-06-24 at 1 50 30 PM](https://user-images.githubusercontent.com/2036909/85607085-c3994800-b621-11ea-9ad5-31df175a4e37.png)

Selected:

![Screen Shot 2020-06-24 at 1 50 57 PM](https://user-images.githubusercontent.com/2036909/85607098-c5fba200-b621-11ea-82c1-eaa3433b41be.png)

#### Testing instructions

- Apply D45434-code to force Giropay to be available.
- Sandbox the store.
- Change your user's currency to `EUR`.
- Add a plan to your cart and visit checkout. 
- Add `?flags=composite-checkout-force` to the query string in the URL and reload the page because currently it is disabled for international currencies.
- Verify that "Giropay" is an option displayed in the final payment step.
- Verify that you cannot pay if the "Name" field is empty.
- Fill out the "Name" field and submit the payment.
- Verify that you are redirected to a Stripe test page.
- Click to accept the payment and verify that the payment completes successfully and that the plan is purchased.
